tests/stubtest_allowlists/py3_common.txt

```diff
@@ -190,9 +190,15 @@ threading.Condition.release  # Condition functions are exported in __init__
 threading.Lock  # A factory function that returns 'most efficient lock'. Marking it as a function will make it harder for users to mark the Lock type.
 tkinter.Misc.grid_propagate  # The noarg placeholder is a set value list
 tkinter.Misc.pack_propagate  # The noarg placeholder is a set value list
+tkinter.Misc.grid_columnconfigure  # Runtime uses **kwargs even though only certain arguments are accepted
```

Comment:
These shouldn't be necessary. stubtest is happy to allow you to spell out arguments instead of kwargs, but if you're doing that then you need to mark them as keyword-only (i.e. `index: type, cnf: type, *, whatever, whatever whatever`)

Reply:
`tkinter.Misc.grid_columnconfigure` is defined as a function, `tkinter.Misc.columnconfigure` (without the `grid_` prefix) is defined as an alias for it.

I added the `*` to mark the keyword-only arguments, and now stubtest is happy with `tkinter.Misc.columnconfigure`, but it still requires an allowlist entry for tkinter.Misc.grid_columnconfigure`, or else it complains that an empty dict is not a suitable default value for an argument of type `TypedDict(total=False)`